lib/plausible/open_telemetry.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Plausible.OpenTelemetry do
+  @moduledoc false
+
+  require OpenTelemetry.Tracer, as: Tracer
+
+  def add_site_attributes(site) do
+    case site do
+      %Plausible.Site{} = user ->
```

Comment:
```suggestion
      %Plausible.Site{} ->
```

PS: we should make sure those warnings make PRs red